src/settlement/ethereum/client.rs

```diff
@@ -0,0 +1,3 @@
+//!
```

Comment:
the settlement should be aggregator. we use aggregator to submit the batch and its proof to L1.